templates/results.html

```diff
@@ -70,8 +70,20 @@
 	<h1><span class="icon icon_${os.title_icon}" />${os.title}</h1>
 
       <div class="body">
-	<div>
 	  <ul class="results">
+        <p>Sort By:</p>
+        <div class="sort-dropdown">
+        <button class="sort-dropdown-toggle" aria-haspopup="true" aria-expanded="false">
+        <span>${'Title' if os.sort_order == 'alpha' else os.sort_order.replace('_', ' ').title()}</span><span>▼</span>
+        </button>
+        <div class="sort-dropdown-menu" role="menu">
+            <py:for each="x in os.sort_orders">
+                <py:if test="os.sort_order != x">
+                    <a href="?sort_order=${x}" role="menuitem">${'Title' if x == 'alpha' else x.replace('_', ' ').title()}</a>
```

Comment:
use the sorted_msgs dict in OpenSearch for this

Reply:
<img width="342" height="237" alt="image" src="https://github.com/user-attachments/assets/7dbe5295-f3bb-4034-b165-40117d845390" />

The sorted_msgs text don't fit quite as well in this application. But happy to update to that if it is preferred. Looks like those might be localized?